### PR TITLE
Updates `BadgeMessageView` to have continuous corner curve on iOS 13

### PR DESCRIPTION
**Pull request by simonmitchell**

### CODE REVIEW

File: MessageStackView/Messages/BadgeMessage/BadgeMessageView.swift [hunk 1]
```diff
@@ -138,6 +138,11 @@ open class BadgeMessageView: UIView {
         containerView.layer.cornerRadius = layer.cornerRadius
         shadow = BadgeMessageView.shadowComponents
         
+        if #available(iOS 13.0, *) {
+            layer.cornerCurve = .continuous
+            containerView.layer.cornerRadius = layer.cornerRadius
```
Comment (BenCShutt) [inline MessageStackView/Messages/BadgeMessage/BadgeMessageView.swift:143]: Can we remove this line as it's done above? Or must it come `layer.cornerCurve = .continuous`

**Comment by BenCShutt** [comment]
Nice find btw! :D Is the change noticeably different would you say? Not that we don't need it, I'm very for it :D

File: MessageStackView/Messages/BadgeMessage/BadgeMessageView.swift (hunk 1 above)
Comment (simonmitchell) [reply MessageStackView/Messages/BadgeMessage/BadgeMessageView.swift:143]: Sorry, meant to also set the cornerCurve!

**Comment by simonmitchell** [comment]
It's subtle, but definitely noticeable, even if subconsciously! ☺️

**Comment by BenCShutt** [comment]
@simonmitchell Very neat :) Maybe something we could do with implementing elsewhere a lot more!

**Pull request merged**